### ISSUE TITLE
Move us onto stable decorators

### ISSUE THREAD

**Issue by ef4** [emberjs/ember.js#20778]
The recent build refactors should have made it easier now to run our tests under both legacy decorators and stable decorators. That would allow us to pick back up https://github.com/emberjs/ember.js/pull/20507

**Comment by andiz2** [comment]
Hi ef4,

Would love to take care of this.
I would like to have more infos if is possible since is my first contribution to this project and might help if I'll have it more clear.

Thanks,
Andiz2

**Comment by ef4** [comment]
Much more progress in https://github.com/emberjs/ember.js/pull/20868

**Comment by ef4** [comment]
The next actions here are
 - update the PR to include typescript support for stable decorators and exercise it in a smokiest app.
 - write the RFC for how people should transition

**Comment by wcr-karan** [comment]
Hi @ef4 @mansona, I’ve implemented the fix for this issue and opened PR #21012.  
Whenever you get a moment, I’d appreciate a review and merge it.Thank you!

**Comment by NullVoxPopuli** [comment]
@wcr-karan you did no such thing. please don't submit prs that delete a ton of our comments, or change our arrays apis.

**Comment by ro0gr** [comment]
@NullVoxPopuli closed by mistake?

**Comment by NullVoxPopuli** [comment]
yes! tyty I closed an AI generated PR, and I don't know why GH thought this was solved. Silly GH.

**Comment by GujjetiMokshith** [comment]
@NullVoxPopuli Is this issue open to work on ?

**Comment by NullVoxPopuli** [comment]
@GujjetiMokshith 
All issues are on all repos: whoever does the most correct implementation first, wins
